### ISSUE TITLE
Loading the model

### ISSUE THREAD

**Issue by billyAlexio** [tom-doerr/carla_rl_weights#2]
<img width="658" alt="screenshot 2023-09-03 233942" src="https://github.com/tom-doerr/carla_rl_weights/assets/80588386/14ec79a7-8a6c-4283-b147-e64f8cd5e1e3">

Hi, may I ask how can I load your pretrained .model file into Keras properly? I use tensorflow 1.13.1, Keras 2.2.4, and it seems there is a problem loading the model. 

I would be very grateful if you can take a look of my problem or tell me your solution!

**Comment by tom-doerr** [comment]
Could it be that you need to select the correct network architecture for the checkpoints in the model_config?

**Comment by billyAlexio** [comment]
<img width="709" alt="screenshot 2023-09-04 111240" src="https://github.com/tom-doerr/carla_rl_weights/assets/80588386/aefe1c33-6ae3-4196-8241-a2828ae1ac34">

Thanks for your reply! I don't think network architecture could be the problem, as I simply import keras and use keras.model.load_model() function, there appears the same error.

**Comment by billyAlexio** [comment]
Thanks for your reply! I don't think network architecture could be the problem, as I simply import keras and use keras.model.load_model() function, there appears the same error.

<img width="709" alt="screenshot 2023-09-04 111240" src="https://github.com/tom-doerr/carla_rl_weights/assets/80588386/e16577d2-5a8b-4644-a518-4d5183fcab94">

**Comment by tom-doerr** [comment]
Could you just remove the decode part? Don't think you need the decode when it's a str.

**Comment by billyAlexio** [comment]
Yes, and I find a solution by reinstalling another version of h5py package, but when I run play.py another problem happened, as shown in the image. I have looked up for this issue, most answers revolves around the older version of keras should use lr rather than learning_rate as the parameter, however, I have checked the original code, sendex did pass lr as the parameter. And when I run train.py, there is no error, but the trained seems to stuck at waiting state. I am a little confused.

<img width="720" alt="screenshot 2023-09-04 121835" src="https://github.com/tom-doerr/carla_rl_weights/assets/80588386/928e6268-4791-41be-88c1-8396b362f693">

**Comment by tom-doerr** [comment]
Maybe ChatGPT could help

**Comment by billyAlexio** [comment]
Yeah maybe, thanks for the tip!
